book/website/project-design/pd-checklist.md

```diff
@@ -0,0 +1,69 @@
+(pd-checklist)=
+# Getting Started Checklist 
+
+We can begin the project design process by identifying different parts of our research, such as main research questions, methods and materials, code and data requirements, workflow, communication channels, ways of working, collaborative practices, and so on. 
+This process allows us to be intentional from the start to ensure that our research is reproducible, well-communicated, and inclusive of all stakeholders where decisions are collaboratively made. 
+We can explore and select the right tools and methods for reproducibility in our research and promote good practices such as documentation, version control, peer-review processes, testing, workflow, archiving, and data management plans from the beginning. 
+Finally, we can plan for publishing and sharing research components before, during, and after the project. 
+
+**Below is a checklist you can use to help identify areas of project planning you might want to look at.**
+
+### Aims & Values 
```

Comment:
I think this change from heading level 1 to level 3 is what is causing the build to fail (this is a warning, but we have strict builds :smile:).

Changing the `###` headings in this file to `##` should fix it, I hope.

Reply:
I was just about to write that as well! :)